src/main/java/HumanPlayer.java

```diff
@@ -0,0 +1,30 @@
+import java.util.List;
+
+public class HumanPlayer implements Player{
+
+    private String name = "";
+    private String marker = "";
+
+    public String getName() {
+        return name;
+    }
+
+    public String getMarker() {
+        return marker;
+    }
+
+    public void build(String name, String marker) {
+        this.name = name;
+        this.marker = marker;
+    }
+
+    public List<String> makeMove(Board currentBoard) {
+        int cellPosition = getCellPosition("1");
```

Comment:
Would it be better for `cellPosition` to be an int? (I assume you'll build out the functionality and not leave the player always taking position 1 😸 )